demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1233,63 +1208,57 @@ static void prvMQTTAgentCmdCompleteCallback( CommandContext_t * pxCommandContext
                      eSetValueWithOverwrite );
     }
 }
-
 /*-----------------------------------------------------------*/
 
-static void prvRegisterOTACallback( const char * pTopicFilter,
-                                    uint16_t topicFilterLength )
+static void prvRegisterOTACallback( const char * pcTopicFilter,
+                                    uint16_t usTopicFilterLength )
 {
     bool isMatch = false;
-    MQTTStatus_t mqttStatus = MQTTSuccess;
-    uint16_t index = 0U;
-    uint16_t numTopicFilters = sizeof( otaTopicFilterCallbacks ) / sizeof( OtaTopicFilterCallback_t );
-
-
     bool subscriptionAdded;
-
-    ( void ) mqttStatus;
+    MQTTStatus_t xMqttStatus = MQTTSuccess;
+    uint16_t usIndex = 0U;
+    uint16_t usNumTopicFilters = sizeof( xOtaTopicFilterCallbacks ) / sizeof( OtaTopicFilterCallback_t );
 
     /* Match the input topic filter against the wild-card pattern of topics filters
     * relevant for the OTA Update service to determine the type of topic filter. */
-    for( ; index < numTopicFilters; index++ )
+    for( ; usIndex < usNumTopicFilters; usIndex++ )
```

Comment:
Can you set usIndex to zero here for readability? I know it's not necessary but the compiler should optimize this out anyways so it shouldn't have any impact on the performance.

Reply:
Done https://github.com/aws/amazon-freertos/pull/3148/commits/6bbe8a3c217e31fff82104a685ad811120104724